google/cloud/bigtable/internal/bulk_mutator_test.cc

```diff
@@ -417,7 +417,7 @@ TEST(MultipleRowsMutatorTest, RetryOnlyIdempotent) {
 
   EXPECT_EQ(2, failures[1].original_index());
   // EXPECT_EQ("baz", failures[1].mutation().row_key());
-  EXPECT_EQ(google::cloud::StatusCode::kUnknown, failures[1].status().code());
+  EXPECT_EQ(google::cloud::StatusCode::kInternal, failures[1].status().code());
```

Comment:
This tests only the first of the changed errors.

Could you please also test the second one? You can easily achieve it by adding another `bt::SingleRowMutation("bar", {bt::SetCell("fam", "col", 0_ms, "qux")}` to the `BulkMutation` in this test and checking that there is one more failure, also with `kInternal`.